static/css/gridpage.css

```diff
@@ -23,6 +23,15 @@
     min-height: 152px;
 }
 
+@media screen and (max-width: 768px){
+    .launch-card {
+        width: 100%;
```

Comment:
:wave: @Arhell .
What do you think about grouping this addition with the existing `@media screen ` entries (which start at line 254)?

Reply:
@kbhawkey  hi!
excellent thank you!
Moved